src/web/assets/htmx/src/htmx.js

```diff
@@ -37,8 +40,11 @@ htmx.defineExtension('craft-cp', {
             }
         }
 
+        return text;
```

Comment:
Maybe we should remove the template tags from the `doc` node and return `doc.textContent` here?